internal/api/attestationconfigapi/version.go

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package attestationconfigapi
+
+import (
+	"encoding/json"
+	"fmt"
+	"path"
+	"sort"
+	"strings"
+
+	"github.com/edgelesssys/constellation/v2/internal/attestation/variant"
+)
+
+// AttestationURLPath is the URL path to the attestation versions.
+const AttestationURLPath = "constellation/v1/attestation"
+
+// SEVSNPVersion tracks the latest version of each component for SEV-SNP.
+type SEVSNPVersion struct {
+	// Bootloader is the latest version of the SEV-SNP bootloader.
+	Bootloader uint8 `json:"bootloader"`
+	// TEE is the latest version of the SEV-SNP TEE.
+	TEE uint8 `json:"tee"`
+	// SNP is the latest version of the SEV-SNP SNP.
+	SNP uint8 `json:"snp"`
+	// Microcode is the latest version of the SEV-SNP microcode.
+	Microcode uint8 `json:"microcode"`
+}
+
+// TDXVersion tracks the latest version of each component for TDX.
+type TDXVersion struct {
+	// QESVN is the latest QE security version number.
+	QESVN uint16 `json:"qeSVN"`
+	// PCESVN is the latest PCE security version number.
+	PCESVN uint16 `json:"pceSVN"`
+	// TEETCBSVN are the latest component-wise security version numbers for the TEE.
+	TEETCBSVN [16]byte `json:"teeTCBSVN"`
+	// QEVendorID is the latest QE vendor ID.
+	QEVendorID [16]byte `json:"qeVendorID"`
+	// XFAM is the latest XFAM field.
+	XFAM [8]byte `json:"xfam"`
+}
+
+// VersionAPIEntry is the request to get the version information of the specific version in the config api.
+//
+// TODO: Because variant is not part of the marshalled JSON, fetcher and client methods need to fill the variant property.
+// In API v2 we should embed the variant in the object and remove some code from fetcher & client.
+// That would remove the possibility of some fetcher/client code forgetting to set the variant.
+type VersionAPIEntry struct {
```

Comment:
I would use a different name here as there already is the versionapi. Which is a distinct thing. Maybe one of `SVNVersion`, `AttestationVersion`, `TCBVersion`, `FirmwareVersion`. Or the plural of those.

Reply:
What about simply going with `Entry`?